src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/NuGet.PackageManagement.PowerShellCmdlets.csproj

```diff
@@ -74,10 +73,22 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="$(NuGetCoreSrcDirectory)NuGet.Resolver\NuGet.Resolver.csproj" />
-    <ProjectReference Include="$(NuGetClientsSrcDirectory)NuGet.PackageManagement.VisualStudio\NuGet.PackageManagement.VisualStudio.csproj" />
-    <ProjectReference Include="$(NuGetClientsSrcDirectory)NuGet.VisualStudio\NuGet.VisualStudio.csproj" />
-    <ProjectReference Include="$(NuGetClientsSrcDirectory)NuGet.VisualStudio.Common\NuGet.VisualStudio.Common.csproj" />
+    <ProjectReference Include="..\..\NuGet.Core\NuGet.Resolver\NuGet.Resolver.csproj">
```

Comment:
This project is legacy CSPROJ and would normally have all this extra metadata.  The usage of a property to locate an item is in my opinion subjective.  Relative paths are just as easy to understand and are only useful to avoid churn if we move projects around.  But our code base would still have plenty of churn if we moved things so I'm not convinced its really doing anything.  

Technically this change was made automatically for me by Visual Studio (I didn't hand edit this) so I think we'll always be fighting Visual Studio if we make this too custom.

Reply:
I agree it's subjective. I tend to find the properties easier mostly because I hate counting `..` :D
Most projects use properties, so that's why I was asking.